Description: By default, AX_CHECK_COMPILE_FLAG overrides the *FLAG, so lets stop doing that.
Author: Gianfranco Costamagna <locutusofborg@debian.org>
Forwarded: https://github.com/BLAKE2/libb2/pull/24
Last-Update: 2019-03-14

--- libb2-0.98.1.orig/configure.ac
+++ libb2-0.98.1/configure.ac
@@ -60,7 +60,7 @@ AC_HELP_STRING([--enable-native],
                [enable_native=yes]
 )
 
-AX_CHECK_COMPILE_FLAG([-O3], [CFLAGS=-O3])
+AX_CHECK_COMPILE_FLAG([-O3], [CFLAGS="$CFLAGS -O3"])
 dnl Not all architectures support -march=native
 if test $enable_native == "yes"; then
   AX_CHECK_COMPILE_FLAG([-march=native], [], [enable_native=no])
